apps/meteor/ee/app/federation-v2/server/infrastructure/rocket-chat/hooks/index.ts

```diff
@@ -18,14 +18,15 @@ export class FederationHooksEE {
 		);
 	}
 
-	public static onUsersAddedToARoom(callback: (room: IRoom, inviter: IUser, addedUsers: IUser[]) => Promise<void>): void {
+	public static onUsersAddedToARoom(callback: (room: IRoom, inviter: IUser, addedUsers: IUser[] | string[]) => Promise<void>): void {
 		callbacks.add(
-			'afterAddedToRoom',
-			(params: { user: IUser; inviter: IUser }, room: IRoom): void => {
+			'federation.afterAddUsersToARoom',
```

Comment:
The old callback was executed in many places that this new one will not:
Just want to confirm if this is intended.

 - User added by invite link
 - User joined through integrations such as IRC and Slack bridges, SAML, LDAP and OAuth.
 - `/invite` slash command
 - Some Teams and Livechat features
 - User manually joined a public room.

Reply:
You are right, I should not just replace them, we still need the "old" one here as well. Tks.